src/Plugin/Validation/Constraint/LowercaseEmailConstraintValidator.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace Drupal\apigee_edge\Plugin\Validation\Constraint;
+
+use Symfony\Component\Validator\Constraint;
+use Symfony\Component\Validator\ConstraintValidator;
+
+/**
+ * Validates the LowercaseEmail constraint.
+ */
+class LowercaseEmailConstraintValidator extends ConstraintValidator {
+
+  /**
+   * {@inheritdoc}
+   */
+  public function validate($value, Constraint $constraint) {
+    foreach ($value as $item) {
+      // First check if the organization is ApigeeX.
+      try {
+        /** @var \Drupal\apigee_edge\SDKConnectorInterface $sdk_connector */
+        $sdk_connector = \Drupal::service('apigee_edge.sdk_connector');
```

Comment:
Dependency injection is possible in constraint validators as well. 

See: `\Drupal\Core\Entity\Plugin\Validation\Constraint\ValidReferenceConstraintValidator`

Reply:
Updated the code. Thanks for the suggestion.